definitions/pipelines/detect_variants_mouse.cwl

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Detect Variants workflow"
+requirements:
+    - class: SubworkflowFeatureRequirement
+inputs:
+    reference:
+        type: string
+    tumor_bam:
+        type: File
+        secondaryFiles: [.bai,^.bai]
+    normal_bam:
+        type: File
+        secondaryFiles: [.bai,^.bai]
+    interval_list:
+        type: File
+    strelka_exome_mode:
+        type: boolean
+    strelka_cpu_reserved:
+        type: int?
+        default: 8
+    readcount_minimum_base_quality:
+        type: int?
+    readcount_minimum_mapping_quality:
+        type: int?
+    mutect_scatter_count:
+        type: int?
+    mutect_artifact_detection_mode:
+        type: boolean?
+    mutect_max_alt_allele_in_normal_fraction:
+        type: float?
+    mutect_max_alt_alleles_in_normal_count:
+        type: int?
+    varscan_strand_filter:
+        type: int?
+        default: 0
+    varscan_min_coverage:
+        type: int?
+        default: 8
+    varscan_min_var_freq:
+        type: float?
+        default: 0.1
+    varscan_p_value:
+        type: float?
+        default: 0.99
+    varscan_max_normal_freq:
+        type: float?
+    pindel_insert_size:
+        type: int
+        default: 400
+    vep_cache_dir:
+        type: string
+    vep_ensembl_assembly:
+        type: string
+        doc: "genome assembly to use in vep. Examples: GRCh38 or GRCm38"
+    vep_ensembl_version:
+        type: string
+        doc: "ensembl version - Must be present in the cache directory. Example: 95"
+    vep_ensembl_species:
+        type: string
+        doc: "ensembl species - Must be present in the cache directory. Examples: homo_sapiens or mus_musculus"
```

Comment:
One of these examples seems more relevant to `detect_variants_mouse.cwl` than the other :smile: